src/In.ProjectEKA.HipService/appsettings.json

```diff
@@ -11,5 +11,9 @@
   },
   "OtpService": {
     "BaseUrl":"http://172.16.2.27:5000"
+  },
+  "rabbit": {
+    "HostName": "localhost",
+    "Port": 5672
   }
```

Comment:
This will not work on dev server no.

Reply:
Yeah it has to be injected by enviroment variable

Reply:
can you rename localhost to rabbitmq?